dask/array/tests/test_array_core.py

```diff
@@ -1781,3 +1781,25 @@ def test_to_imperative():
 
     [[a, b], [c, d]] = y.to_imperative()
     assert eq(a.compute(), y[:2, :2])
+
+
+def test_tril_triu():
+    A = np.random.randn(20, 20)
+    for chunk in [5, 7]:
+        dA = da.from_array(A, (chunk, chunk))
+
+        assert np.allclose(da.triu(dA).compute(), np.triu(A))
+        assert np.allclose(da.tril(dA).compute(), np.tril(A))
+
+        for k in range(-25, 25):
+            assert np.allclose(da.triu(dA, k).compute(), np.triu(A, k))
+            assert np.allclose(da.tril(dA, k).compute(), np.tril(A, k))
```

Comment:
Are all of these 100 cases necessary to ensure correctness?  Is there a smaller set that would do as well?

It is useful to keep test times fast if possible.